Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -1459,20 +1414,8 @@ WarpXOpenPMDPlot::WriteOpenPMDFieldsAll ( //const std::string& filename,
 #endif
                 {
                     amrex::Real const *local_data = fab.dataPtr(icomp);
-#ifdef WARPX_DIM_RZ
```

Comment:
Note: this block was only taking the CPU path, and we forgot to transpose on GPU. With this PR, we don't need to anymore anyway :)